quickjs/src/jniMain/kotlin/app/cash/quickjs/ZiplineJvm.kt

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2021 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.quickjs
+
+import app.cash.quickjs.internal.bridge.InboundService
+import app.cash.quickjs.internal.bridge.InternalBridge
+import app.cash.quickjs.internal.bridge.KtBridge
+import app.cash.quickjs.internal.bridge.OutboundClientFactory
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.delay
+
+internal class ZiplineJvm(
+  override val quickJs: QuickJs,
+  override val dispatcher: CoroutineDispatcher,
+) : Zipline(), HostPlatform {
+  private val ktBridge: KtBridge
+  private val jsPlatform: JsPlatform
+
+  init {
+    val jsInboundBridge = object : InternalBridge {
```

Comment:
Implement this on the `ZipelineJvm` directly

Reply:
Good call. Done.